### ISSUE TITLE
New Label needed on cf-system

### ISSUE THREAD

**Issue by kkburr** [cloudfoundry/cf-for-k8s#261]
**Is your feature request related to a problem? Please describe.**
When we have installed prometheus onto our cluster using helm, it currently cannot scrape any metrics from "cf pushed" app pods in the cf-workloads namespace because of the network policies in place that restrict ingress:

https://github.com/cloudfoundry/cf-for-k8s/blob/d9cf7e018d84bcf07088c94e86a1bf7cada6b253/config/eirini.yml#L26
https://github.com/cloudfoundry/cf-for-k8s/blob/d9cf7e018d84bcf07088c94e86a1bf7cada6b253/config/_ytt_lib/eirini/rendered.yml#L5

**Describe the solution you'd like**
After talking to @tcdowney, we agreed that the best solution would be to create a network policy that allows ingress from prometheus to a pod in cf-workloads. That requires 3 things:

- the new network policy (we will provide an example in our [examples repo](https://github.com/cloudfoundry/cf-for-k8s-metric-examples/tree/master)...currently on a branch to go into master today)
- prometheus to be installed with the new pod label expected in the network policy (and installed in the cf-system namespace since it has istio enabled already)
- the cf-system namespace to have the new namespace label expected in the network policy.

We need you all to take care of bullet point 3 by adding this label to the `cf-system` namespace:
`cf-for-k8s.cloudfoundry.org/cf-system-ns: ""`

Please see the thread with additional context: https://cloudfoundry.slack.com/archives/CH9LF6V1P/p1593188901246400

**Describe alternatives you've considered**
We considered using existing labels defined in the network policies above ^ that would allow our traffic in, but this was a hack solution.

CC @heycait

**Comment by kkburr** [comment]
We decided to change our network policy to match on `name: cf-system`:
```
---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: allow-prometheus-scrape
  namespace: cf-workloads
spec:
  podSelector: {}
  policyTypes:
  - Ingress
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          name: cf-system
      podSelector:
        matchLabels:
          what-am-i: prometheus
```

**Comment by jamespollard8** [comment]
Fixed by fa7c634
